### PR TITLE
Add UI PR

### PR DESCRIPTION
**Pull request by AmaduKamara**
# OOP school library: add basic UI

### Objectives completed:
- Build interactive console apps.
- Run a program using the command line.

👨‍💻 Completed the following tasks:
- [ ] List all books.
- [ ] List all people.
- [ ] Create persons.
- [ ] Create a book.
- [ ] Create a rental.
- [ ] List all rentals for a given person id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    start_app
+  end
+
+  def start_app
+    puts "\nPlease choose option by entering a number: "
+    puts '1. List all books.'
+    puts '2. List all people.'
+    puts '3. Create a person'
+    puts '4. Create a book.'
+    puts '5. Create a rental.'
```
Comment (omar-labana) [inline app.rb:20]: - [x] After entering the date for a rental the app crashes with this error msg:
image

**Review by omar-labana — COMMENTED** [review]
Hi @AmaduKamara, 

Excellent work so far! 👌
You have implemented almost all the required features correctly. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (AmaduKamara) [reply app.rb:20]: Hi @omar-labana , Thanks for your review.
I ran the app to check and see if I'll get the same error but everything work fine.
Here is a screenshot of the app running successfully
image

Comment (AmaduKamara) [reply app.rb:20]: This is another screenshot
image

Comment (rloterh) [reply app.rb:20]: @AmaduKamara 

The original example from the videos used numbers as dates, and meanwhile, dates can be written in various formats. It's therefore very appropriate to leave a note in your PR or ReadMe of this project on how can be run, or on how exactly the date should be entered.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amkam,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Ap is running as intended
- No linter errors


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**